src/Arcus.Scripting.ApiManagement/Scripts/Create-AzApiManagementUserAccountsFromConfig.ps1

```diff
@@ -0,0 +1,215 @@
+param(
+    [string][Parameter(Mandatory = $true)] $ResourceGroupName = $(throw "Resource group name is required"),
+    [string][parameter(Mandatory = $true)] $ServiceName = $(throw "API management service name is required"),
+    [string][Parameter(Mandatory = $true)] $ConfigurationFile = $(throw "Path to the configuration file is required"),
+    [switch][parameter(Mandatory = $false)] $StrictlyFollowConfigurationFile = $false,
+    [string][parameter(Mandatory = $false)] $ApiVersion = "2021-08-01",
+    [string][parameter(Mandatory = $false)] $SubscriptionId,
+    [string][parameter(Mandatory = $false)] $AccessToken
+)
+
+if (-not (Test-Path -Path $ConfigurationFile)) {
+    throw "Cannot apply user configuration based on JSON configuration file because no file was found at: '$ConfigurationFile'"
+}
+if ((Get-Content -Path $ConfigurationFile -Raw) -eq $null) {
+    throw "Cannot apply user configuration based on JSON configuration file because the file is empty."
+}
+
+$schema = @'
```

Comment:
Could this be an external file we read here? Would possably improve readability + would notify us if we have a problem with the JSON structure.

Reply:
For now I would keep it as it is now, we also use this [here](https://github.com/arcus-azure/arcus.scripting/blob/main/src/Arcus.Scripting.Storage.Table/Scripts/Set-AzTableStorageEntities.ps1).
Maybe we could look at this later on as an enhancement and implement this in both scripts 👍🏻

Reply:
Well, yes, but the example in for the Table storage script is a lot smaller than this one. That's a bit why I suggested it.
Okay for a follow-up PR. Can you create an issue for that and link here? Thx!

Reply:
Added an issue to track this: https://github.com/arcus-azure/arcus.scripting/issues/423